### PR TITLE
Docs: Update new Browse Dashboards

### PR DESCRIPTION
**Pull request by joshhunt**
Updates the Browse Dashboards Whats New to:

 - make it a bit more user-focused
 - mention that it's now generally available

### CODE REVIEW

File: docs/sources/whatsnew/whats-new-next/index.md [hunk 1]
```diff
@@ -136,14 +136,16 @@ Use the Grafana Alerting - Grafana OnCall integration to effortlessly connect al
 
 To learn more, refer to the [Grafana OnCall integration for Alerting documentation](/docs/grafana/next/alerting/alerting-rules/manage-contact-points/configure-oncall/).
 
-## New browse dashboards view
+## New Browse Dashboards
```
Comment (imatwawana) [inline docs/sources/whatsnew/whats-new-next/index.md:139]: I'm assuming that we're talking about browsing dashboards in general, so this should be lower case
```suggestion
## New browse dashboards
```

File: docs/sources/whatsnew/whats-new-next/index.md [hunk 2]
```diff
@@ -136,14 +136,16 @@ Use the Grafana Alerting - Grafana OnCall integration to effortlessly connect al
...
 
 <!-- Yaelle Chaudy for Frontend Platform -->
 <!-- OSS, Enterprise -->
```
Comment (imatwawana) [inline docs/sources/whatsnew/whats-new-next/index.md:143]: We want to add the feature release dates here so can you please add that.
 ```suggestion

<Date>

```

File: docs/sources/whatsnew/whats-new-next/index.md [hunk 3]
```diff
@@ -136,14 +136,16 @@ Use the Grafana Alerting - Grafana OnCall integration to effortlessly connect al
...
 
-_Available in public preview in Grafana Cloud_
+_Generally available in Grafana Cloud_
+
+The new Browse Dashboards interface features a more compact design, making it easier to navigate, search for, and manage for your folders and dashboards. The new interface also has many performance improvements, especially for instances with a large number of folders and dashboards.
```
Comment (imatwawana) [inline docs/sources/whatsnew/whats-new-next/index.md:146]: I don't see anything in the UI with this name and this isn't a product name, so it should be lower case
```suggestion
The new browse dashboards interface features a more compact design, making it easier to navigate, search for, and manage for your folders and dashboards. The new interface also has many performance improvements, especially for instances with a large number of folders and dashboards.
```

**Review by imatwawana — COMMENTED** [review]
A few styling issues and one field we'd like to add, but other than that it's good to go!

**Review by imatwawana — APPROVED** [review]
LGTM!

**Comment by joshhunt** [comment]
@imatwawana will this need to be added to the 10.2 whats new also, or has that not been created yet?

**Comment by imatwawana** [comment]
> @imatwawana will this need to be added to the 10.2 whats new also, or has that not been created yet?

I'll be adding it as part of a round up of items in the Cloud what's new that need to also be in the on-prem. I'll tag you when that's ready for review.

**Pull request merged**